lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_builtin_method.rb

```diff
@@ -34,7 +34,7 @@ def self.miq_log_workspace(obj, _inputs)
     end
 
     def self.miq_send_email(_obj, inputs)
-      MiqAeMethodService::MiqAeServiceMethods.send_email(inputs["to"], inputs["from"], inputs["subject"], inputs["body"])
+      MiqAeMethodService::MiqAeServiceMethods.send_email(inputs["to"], inputs["from"], inputs["cc"], inputs["bcc"], inputs["subject"], inputs["body"])
```

Comment:
This seems not matching ```send_email``` method signature.

Reply:
Do we ever use the optional content_type?  Is this the only caller?